cms/topics/models.py

```diff
@@ -222,6 +231,26 @@ def processed_methodologies(self) -> list[MethodologyPage]:
         )
         return highlighted_pages + pages
 
+    @cached_property
+    def formatted_datasets(self) -> list[dict[str, Any]]:
```

Comment:
I see we do something similar here:

https://github.com/ONSdigital/dis-wagtail/blob/ed64a48bae7621c9b2a319e63134d9d18b2a73d8/cms/articles/models.py#L386

I wonder if it's worth abstracting this somehow? This is a non-blocking suggestion.

Reply:
[a737d55](https://github.com/ONSdigital/dis-wagtail/pull/226/commits/a737d55fc0054ca9ff8e116ec649f80803f0a117)